kolibri/plugins/media_player/assets/src/views/MediaPlayerIndex.vue

```diff
@@ -625,7 +636,7 @@
   }
   .wrapper.transcript-wrap .media-player-transcript {
     left: 0;
-    height: $transcript-wrap-height;
+    height: calc(#{$transcript-wrap-height});
```

Comment:
Minor but could this calc happen at variable definition?

Reply:
Unfortunately, it can't, since this relies on width and height of the viewport. The width affects the video height, which in turn is subtracted from the viewport height to determine this height